docs/sources/mimir/get-started/play-with-grafana-mimir/index.md

```diff
@@ -137,22 +185,28 @@ rule is now being created in Grafana Mimir ruler and will be soon available for
    ```
 1. Confirm the query returns a value of `3` which is the number of Mimir instances currently running in your local setup.
 
+<!-- INTERACTIVE page step4.md END -->
+
+<!-- INTERACTIVE page step5.md START -->
+
 ## Configure your first alert rule
 
 Alerting rules allow you to define alert conditions based on PromQL expressions and to send notifications about firing
 alerts to Grafana Mimir Alertmanager. In this section you're going to configure an alerting rule in Grafana Mimir using
 tooling offered by Grafana.
 
 1. Open [Grafana Alerting](http://localhost:9000/alerting/list).
-2. Click **New alert rule**.
-3. Configure the alert rule:
-   1. Select **Mimir or Loki alert** in the top selector.
-   2. Choose **Mimir** in the **Select data source** field.
-   3. Type `up == 0` in the **Metrics browser** query field.
-   4. Type `MimirNotRunning` in the **Rule name** field.
-   5. Select `example-namespace` in the **Namespace** field.
-   6. Select `example-group` in the **Group** field.
-   7. From the upper-right corner, click the **Save and exit** button.
+1. Click **New alert rule**.
+1. Configure the alert rule:
+   1. Type `MimirNotRunning` in the **Rule name** field.
+   1. Choose **Mimir** in the **Select data source** field.
+   1. Type `count(up == 0)` in the **Metrics browser** query field. This will currently show `no data` since all instances are running.
```

Comment:
```suggestion
   1. Type `count(up == 0)` in the **Metrics browser** query field. This currently shows `no data` since all instances are running.
```